I ran into an issue with the negative values. No matter what I tried, I got similar errors:

`py wc.py -i ff3.smc -o tests/test_neg_min.sfc -si 230.-2.2` fails during generation:
```
PS C:\Projects\WorldsCollide> py wc.py -i ff3.smc -o tests/test_neg_min.sfc -si 230.-2.2
Version   1.4.3d
Generated 2026-05-06 11:43:27
Input     ff3.smc
Output    test_neg_min.sfc
Log       test_neg_min.txt
Seed      gxys1vqrl13m
Flags     -drloc shuffle -stloc shuffle -si 230.-2.2 -ir standard -chrm 0 0
Hash      Ultros, Yura, Narshe Guard, Maduin
Traceback (most recent call last):
  File "C:\Projects\WorldsCollide\wc.py", line 30, in <module>
    main()
    ~~~~^^
  File "C:\Projects\WorldsCollide\wc.py", line 27, in main
    memory.write()
    ~~~~~~~~~~~~^^
  File "C:\Projects\WorldsCollide\memory\memory.py", line 14, in write
    self.rom.write(args.output_file)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\Projects\WorldsCollide\memory\rom.py", line 24, in write
    out_file.write(bytearray(self.data))
                   ~~~~~~~~~^^^^^^^^^^^
ValueError: byte must be in range(0, 256)
```

Also, there's no `sys.exit(1)` call after the min/max eval like the others - that should be added to be consistent

```
            if max < min:
                import sys
                args.parser.print_usage()
                print(f"{sys.argv[0]}: error: start-items: max:'{max}' must be greater than or equal to the min:'{min}'")
                sys.exit(1)
```

--- a/args/starting_gold_items.py
+++ b/args/starting_gold_items.py
@@ -100,6 +100,7 @@ def process(args):
                 import sys
                 args.parser.print_usage()
                 print(f"{sys.argv[0]}: error: start-items: max:'{max}' must be greater than or equal to the min:'{min}'")
+                sys.exit(1)
 
             item = StartingItem(item_id, min, max)
             args.start_items_list.append(item)
@@ -146,10 +147,13 @@ def options(args):
     for item in args.start_items_list:
         from constants.items import id_name
         item_name = id_name[item.id]
+        min = item.min
+        if min < 0:
+            min = 0
         if not item_name.endswith("s"):
             item_name = item_name + "s"
         opts += [
-            (f"Start {item_name}", f"{item.min}-{item.max}", "start_items")
+            (f"Start {item_name}", f"{min}-{item.max}", "start_items")
         ]
 
     return opts

--- a/event/start.py
+++ b/event/start.py
@@ -215,7 +215,7 @@ class Start(Event):
                 continue
 
             src += [
-                field.AddItems(item.id, item_count, sound_effect = False),
+                field.AddItems(item.id, item_count[0], sound_effect = False),
             ]
 
         if self.args.debug:

--- a/event/start.py
+++ b/event/start.py
@@ -211,6 +211,8 @@ class Start(Event):
 
         for item in self.args.start_items_list:
             item_count = random.sample(range(item.min, item.max+1), 1)
+            if item_count[0] < 0:
+                item_count[0] = 0
             src += [
                 field.AddItems(item.id, item_count, sound_effect = False),
             ]

--- a/event/start.py
+++ b/event/start.py
@@ -211,8 +211,9 @@ class Start(Event):
 
         for item in self.args.start_items_list:
             item_count = random.sample(range(item.min, item.max+1), 1)
-            if item_count[0] < 0:
-                item_count[0] = 0
+            if item_count[0] <= 0:
+                continue
+
             src += [
                 field.AddItems(item.id, item_count, sound_effect = False),
             ]

